transforms/acceptance/index.js

```diff
@@ -18,34 +14,32 @@ module.exports = function transformer(file, api) {
 
   // fetching store
   code = removeImport(j, code, 'embercom/lib/container-lookup');
-  code = findCallExpression(j, code, "getEmberDataStore")
+  code = findCallExpression(j, code, 'getEmberDataStore')
     .forEach(path => {
-      j(path).replaceWith(j.identifier("this.owner.lookup('service:store')"))
+      j(path).replaceWith(j.identifier("this.owner.lookup('service:store')"));
     })
     .toSource();
 
   // register and inject helpers
-  code = j(file.source)
-  .find(j.MemberExpression, {
-    object: {
+  code = j(code)
```

Comment:
This transform was operating on `file.source` which meant that all previous codemods that were already stored in `code` where lost.